packages/terra-embedded-component/src/EmbeddedComponent.jsx

```diff
@@ -1,31 +1,135 @@
+/* global SystemJS */
 import React from 'react';
 import PropTypes from 'prop-types';
-import classNames from 'classnames/bind';
-import 'terra-base/lib/baseStyles';
-import styles from './EmbeddedComponent.scss';
+import axios from 'axios';
+import 'systemjs/dist/system-production';
+import { loadStylesheet } from './assets';
 
-const cx = classNames.bind(styles);
+// Expose AMD required globals on the window
+window.define = window.SystemJS.amdDefine;
+window.require = window.requirejs = SystemJS.amdRequire;
 
 const propTypes = {
- /*
- * Content to be displayed as the name
- */
-  name: PropTypes.string,
+
+  /**
+   * The basePath to load the manifest and manifest asset dependencies from.
+   * */
+  basePath: PropTypes.string.isRequired,
+
+  /**
+   * A placeholder to display while the component is loading.
+   * */
+  placeholder: PropTypes.element,
+
+  /**
+   * The number of milliseconds to wait before timing out.
+   */
+  timeout: PropTypes.number,
+
+  /**
+   * The entry file in the manifest to use to load the embeddable module.
+   * */
+  entry: PropTypes.string,
+
+  /**
+   * A manifest object defining the asset dependencies. If provided will be used instead
+   * of fetching a manifest from the basePath.
+   * */
+  manifest: PropTypes.object,
 };
 
 const defaultProps = {
-  name: 'default',
+  entry: 'index.js',
+  placeholder: null,
```

Comment:
don't need to specify a default for null

Reply:
Strangely, for some reason the component crashes when the placeholder is not set to null as a default if the placeholder is not sent in the props